vnext/Scripts/Microsoft.ReactNative.Managed.nuspec

```diff
@@ -15,7 +15,7 @@
     <tags>react react-native react-native-windows native-module microsoft c# csharp</tags>
     <dependencies>
       <group targetFramework="UAP10.0">
-        <dependency id="Microsoft.UI.Xaml" version="2.7.0" />
+        <dependency id="Microsoft.UI.Xaml" version="2.8.0" />
```

Comment:
I don't think this needs to be updated; this is the nuget dependency, so C# apps that use the M.RN nuget package need a version of WinUI compatible with what we put on here. 2.7.0 is probably too high of a requirement already; the version we put on here will be interpreted as "this version or higher". See https://docs.microsoft.com/en-us/nuget/reference/nuspec#dependencies-element